database/changesets/20210507_AddWatchersAndLabelsToWorkload.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="chengche@broadinstitute.org"
+               dbms="postgresql"
+               id="20210507"
+               logicalFilePath="20210507_AddColumnsToWorkload.xml"
+               runInTransaction="false">
+        <comment>
+            Add a few new columns to the `workload` table
+        </comment>
+        <addColumn tableName="workload" >
+            <column name="labels"
+                    type="text[]"/>
+            <column name="watchers"
+                    type="text[]" >
```

Comment:
Why `text[]` here?

Reply:
Thanks. Please ensure they have default values of `'{}'`.